content/docs/use-cases/experiment-tracking.md

```diff
@@ -0,0 +1,101 @@
+│ 7317bc6                 │ Jul 18, 2021 │       - │      - │ 10           │ 16               │
+│ ├── e2647ef [exp-ee8a4] │ 05:14 PM     │ 0.23146 │ 0.9145 │ 10           │ 64               │
+│ ├── 15c9451 [exp-a9be6] │ 05:14 PM     │ 0.25231 │ 0.9102 │ 10           │ 32               │
+│ ├── 9c32227 [exp-17dd9] │ 04:46 PM     │ 0.23687 │ 0.9167 │ 10           │ 256              │
+│ ├── 8a9cb15 [exp-29d93] │ 04:46 PM     │ 0.24459 │ 0.9134 │ 10           │ 128              │
+└─────────────────────────┴──────────────┴─────────┴────────┴──────────────┴──────────────────┘
+```
+
+![]() _Organized experiments in DVC_
+
+> 📖 Ready to dive in? See [Get Started: Experiments](/doc/start/experiments).
+
+Unlike tools that focus on experiment navigation, DVC's approach guarantees
+reproducibility by working on top of Git, instead of on being a side system with
+fragile links to Git. This means that your [data is
+tracked][versioning everything] from within project versions (without storing it
+in Git). It also enables distributed collaboration along the way, via optional
+hosting like GitHub or GitLab.
+
+DVC projects provide a seamless, local-first experience (no need for running
+servers, special services, or loading websites). You control where and how your
+data is saved and shared. DVC also improves storage efficiency via
+<abbr>caching</abbr>, preventing repetitive data transfers on every experiment.
+
+On top of all that, DVC is completely language agnostic. You can expect the same
+convenience and performance whether you're using Jupyter Notebooks or Scala, CSV
+data frames or HDFS.
+
+|                       | DVC Experiments                         | MLFlow                                         | Weights & Biases          | TensorBoard             |
+| --------------------- | --------------------------------------- | ---------------------------------------------- | ------------------------- | ----------------------- |
+| Experiment definition | `dvc.yaml` metafiles                    | Implicit (in code)                             | Implicit                  | Implicit                |
+| How to run it         | `dvc exp run` any terminal command      | Code integration                               | Code (Python only)        | Code (Python)           |
+| Data storage          | [User][storage] (yours e.g. cloud)      | User (yours)                                   | Hosted (theirs)           | Limited hosted (theirs) |
+| Compute               | User (e.g [CI/CD][cml])                 | User or [hosted][dbricks]                      | User (e.g. local)         | User (local)            |
+| What can be tracked   | code & data, params, metrics, artifacts | data, params, metrics                          | data, params, environment | params and metrics      |
+| Visualization         | Terminal + [web][dvc studio]            | Web app                                        | Website                   | Website                 |
```

Comment:
- i thought we're not mentioning studio in the table?
- what's the difference between "web," "Web app," and "Website?"

Reply:
This is a new table approach so Studio and CML got back in but could be removed if it's noisy.

Would prefer more general feedback on the overall table structure BTW but thanks for the detailed notes. Which ever one will need a bit of polishing for sure.

Reply:
> what's the difference between "web," "Web app," and "Website?"

Updated for consistency in the latest table BTW. Kept "web app" separate since in plain MLFlow you run the web server locally yourself (access localhost:...)